include/motor-control/core/stepper_motor/tmc2130.hpp

```diff
@@ -446,6 +446,31 @@ struct __attribute__((packed, __may_alias__)) DriveStatus {
     uint32_t stst : 1 = 0;
 };
 
+/**
+ * This register sets the control current for voltage PWM mode stealth chop.
+ */
+struct __attribute__((packed, __may_alias__)) StealthChop {
+    static constexpr Registers address = Registers::PWMCONF;
+    static constexpr bool writable = true;
+    static constexpr uint32_t value_mask = (1 << 22) - 1;
+
+    uint32_t pwm_ampl : 8 = 0;
+    uint32_t pwm_grad : 8 = 0;
+    uint32_t pwm_freq0 : 1 = 0;
```

Comment:
`pwm_freq` and `freewheel` could each be consolidated into 2-bit bitfields, which is how the rest of the registers are defined - I think it makes it a little clearer to set/read registers that way.